src/Widgets/SpaceWidget/SpaceWidgetEditingConfig.js

```diff
@@ -0,0 +1,16 @@
+import * as Scrivito from "scrivito";
+import spaceWidgetIcon from "../../assets/images/space_widget.svg";
+
+Scrivito.provideEditingConfig("SpaceWidget", {
+  title: "Space",
+  thumbnail: spaceWidgetIcon,
+  attributes: {
+    size: {
+      title: "Choose size",
```

Comment:
I think, we should add a description, saying, that one should input a number or something like it. Maybe Andreas can help with that.